### ISSUE TITLE
(Feature Request)s Ability to turn off Autoplay and Recently viewed channels grouping.

### ISSUE THREAD

**Issue by jonm66** [yuliskov/SmartTubeNext#134]
Ability to turn off Autoplay and Recently viewed channels grouping.
I also like the dot on the icon to let you know they have new content.

Thank you.

**Comment by foreveryoungforty** [comment]
Turn off autoplay is already implemented in the player UI with the symbol between next and video speed.... there are a few options with that "arrows"-symbol. Give it a try.

**Comment by Jonzo1111** [comment]
Thank you, I don't know how I missed that.

**Comment by screwyluie** [comment]
yeah I had to come searching here for it as well. seems like a setting that should be in the settings menu... it's odd to think of the UI overlay for a single video as the place to setup global settings.

but at least it's there

**Comment by StormPooper** [comment]
I thought that was a repeat button so didn't touch it, glad I came upon this thread.

**Comment by bill-mcgonigle** [comment]
> it's odd to think of the UI overlay for a single video as the place to setup global settings.

This observation is on point. Would it be possible to also expose the setting in the UI or Video player preferences?

I too searched all the menus before coming to Github before learning to set it on a video's overlay.

It's the only thing that surprised me in an otherwise delightful setup experience.

**Comment by ACR-Jeff** [comment]
Stop playback after one video has been implemented in current latest releases

You have 2 options.
1. As foreveryoungforty mentioned:
> Turn off autoplay is already implemented in the player UI with the symbol between next and video speed.... there are a few options with that "arrows"-symbol. Give it a try.

2. Go to `Settings > Player > Playback mode > Stop playback after one video (except queue)`
